### DISCUSSION TITLE
"database disk image is malformed"

### DISCUSSION THREAD

**Discussion by Ushiroooo** [ppy/osu#14006] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/87917522/126883252-5169658d-5969-4dea-9a1f-549016e2fd74.png)

**Comment by frenzibyte** [comment, marked as answer]
Your database has been corrupted, delete it (stored at `%appdata%/osu/client.db`) and start the game and import again.

**Reply by Ushiroooo** [reply]
It worked, thank you very much! :D
